rnef.config.mjs

```diff
@@ -0,0 +1,13 @@
+import {platformAndroid} from '@rnef/platform-android';
```

Comment:
It looks like (surprisingly?) this is our first `.mjs` file. Can you please ensure that it's being linted by `ESLint` and `Prettier`?

Reply:
ok, resolving